run-checks

```diff
@@ -1,63 +1,94 @@
 #!/bin/sh
 
-set -e
+set -eu
 
 if which goctest >/dev/null; then
     goctest="goctest"
 else
     goctest="go test"
 fi
 
-echo Checking formatting
-fmt=$(gofmt -l .)
-
-if [ -n "$fmt" ]; then
-    echo "Formatting wrong in following files"
-    echo $fmt
-    exit 1
+STATIC=""
+UNIT_GO=""
+UNIT_NODE=""
```

Comment:
Up for debate wether this is called `UNIT_NODE` or `UNIT_JS` as the tests to run are JavaScript unit tests